discovery/marathon/marathon.go

```diff
@@ -45,6 +46,11 @@ const (
 	// taskLabel contains the mesos task name of the app instance.
 	taskLabel model.LabelName = metaLabelPrefix + "task"
 
+	// portMappingsLabelPrefix is the prefix for the application portMappings labels.
+	portMappingsLabelPrefix = metaLabelPrefix + "port_mappings_label_"
```

Comment:
@robsonpeixoto I just took a final look and compared the naming to existing meta labels. It turns out we only use the singular so far, so it should be `..._port_mapping_label_...` etc. I think this make sense, as the label is only attached to a single port mapping, and not to all of them.